WcaOnRails/app/models/competition_event.rb

```diff
@@ -21,4 +21,18 @@ def to_wcif
       "rounds" => self.rounds.map(&:to_wcif),
     }
   end
+
+  def load_wcif!(wcif)
+    self.rounds.each(&:destroy!)
```

Comment:
@jfly actually I suggested it in this PR [some time before](https://github.com/thewca/worldcubeassociation.org/pull/1619#discussion_r116047709) (but I can be missing that the same applied for one of my PRs in the past).